### DISCUSSION TITLE
Fatal error detected

### DISCUSSION THREAD

**Discussion by patiopat** [artisan-roaster-scope/artisan#743] [General]
Hello I am completely new to GitHub and this community so forgive me if I havent posted this in the right section or there is a better place to ask this question.  When I try and start Artisan I get an error that says...Failed to run artisan script.  I am using the latest version of Artisan and have been using it on my Hottop 2k+  I have used the software for montsh without any problems.  I have reinstalled the original program and even downloaded 2.1.1 to see if that makes any difference.  As far as I know there have been no changes to the computer setup or program changes.  I am using windows 10 on a new computer any help would be appreciated.
Pat

**Comment by roasterdave** [comment]
Sounds like either an incomplete installation or not enough user privileges when you run the app .  You did not say what version you are trying to install so please download and work with [v2.4.6](https://github.com/artisan-roaster-scope/artisan/releases/tag/v2.4.6).  Unzip the download file then right click on Setup-Artisan-2.4.6.0.exe and choose Run as Administrator.  Once the install is complete navigate to c:\Program Files\Artisan.  Right click on artisan.exe and again choose Run as Administrator.

**Reply by patiopat** [reply]
Thank you for your help so far.  I was using 2.4.6 the Sweet Maria's sponsored version (many thanks to them too)  I tried unzipping and reinstalling the program from the original file I downloaded a few months ago and think I followed your instructions but the same error code came up.  I am going to try and download it again but am having difficulty downloading it because I am in a remote part of Canada and have no wifi and bad cell/data connections.   Thanks again for your quick response
Pat

**Reply by roasterdave** [reply]
If you still have the downloaded zip file for v2.4.6 you should not need to download it again.  Just try installing and running Artisan with Run as Administrator to see if that works.

**Reply by patiopat** [reply]
I tried reinstalling the older downloaded file and the new downloaded file same error came back up both times.  Since I am only using the computer for Artisan I did a complete system reset,  reinstalled windows and a complete wipe of all data and programs.  I just downloaded the 2.4.6v again installed and opened it as you said and the same fatal error message came up again.  I thought the problem might have been a configuration or system issue with windows so that is why i did a complete factory reset of the computer.  
Any suggestions or help would be greatly appreciated. 
Thanks Pat

**Reply by poundy** [reply]
Just to be certain, you did download the zip file called Artisan-Win-2.4.6.zip from https://github.com/artisan-roaster-scope/artisan/releases/download/v2.4.6/artisan-win-2.4.6.zip, and not the full github code base ? 

Was there something that prompted you to think that Windows was corrupted in some way, leading you to reinstall from scratch? Like you had something else on it that went wrong, or malware, or hardware issues?  That's usually a very large undertaking (and not often needed).

Can you now document your steps closely for us - I'd suggest finding the "programs and features" (classic) control panel item and confirming what apps have been installed there for us (screenshot will be OK). At a minimum I'm expecting to see Artisan, and Python (or multiple Python apps). Then, uninstall them all, then confirm they are all gone (or not - just confirm current state). Then run the reinstall of Artisan, and confirm that the installer runs through to completion without error. Re-confirm what gets added in the programs and features list, and then attempt to start Artisan... And please capture a screenshot of the first error that appears.
